bindings/python/test/geometry/d2/objects/test_composite.py

```diff
@@ -0,0 +1,176 @@
+################################################################################################################################################################
+
+# @project        Open Space Toolkit ▸ Mathematics
+# @file           bindings/python/test/geometry/d2/objects/test_composite.py
+# @author         Remy Derollez <remy@loftorbital.com>
+# @license        Apache License 2.0
+
+################################################################################################################################################################
+
+import pytest
+
+import ostk.mathematics as mathematics
+
+################################################################################################################################################################
+
+Object = mathematics.geometry.d2.Object
+Point = mathematics.geometry.d2.objects.Point
+Polygon = mathematics.geometry.d2.objects.Polygon
+Composite = mathematics.geometry.d2.objects.Composite
+Segment = mathematics.geometry.d2.objects.Segment
+
+################################################################################################################################################################
+
+@pytest.fixture
+def polygon () -> Polygon:
+
+    point_1: Point = Point(-1.0, 1.0)
+    point_2: Point = Point(1.0, 1.0)
+    point_3: Point = Point(1.0, -1.0)
+    point_4: Point = Point(-1.0, -1.0)
+
+    return Polygon([point_1, point_2, point_3, point_4])
+
+@pytest.fixture
+def point () -> Point:
+
+    return Point(-2.0, 5.0)
+
+@pytest.fixture
+def composite (polygon: Polygon, point: Point) -> Composite:
+
+    composite = Composite(polygon)
+    composite += Composite(point)
+
+    return composite
+
+################################################################################################################################################################
+
+class TestComposite:
+
+    def test_geometry_d2_objects_composite_constructor (
```

Comment:
No need for `geometry_d2_objects_composite` since implied by file path + class name; and add `_success` suffix to indicate this is a success-case test.

```suggestion
    def test_constructor_success (
```